virttest/utils_misc.py

```diff
@@ -2374,7 +2376,8 @@ def compare_qemu_version(major, minor, update, is_rhev=True, params={}):
     if installed_version is None:
         logging.error("Cannot get local qemu version, return False directly.")
         return False
-    if is_rhev != installed_version['is_rhev']:
+    if not ((is_rhev and installed_version['is_rhev']) or
+            (is_ma and installed_version['is_ma'])):
```

Comment:
If we want to compare qemu-kvm (not ma, not rhev) builds, that would be:
is_rhev = False
installed_version['is_rhev'] = False
is_ma = False 
installed_version['is_ma'] = False
And here the statement will be ' not ((False and False) or (False and False))' = True, and we'll return a False directly.

Reply:
Could this be:
def compare_qemu_version(major, minor, update, **is_rhev=False, is_ma=False**, params{}:
...
if not ((is_rhev == installed_version['is_rhev']) and (is_ma == installed_version['is_ma'])):
    return False

Reply:
@lento-sun Thx, done